website/pages/docs/plugins/sources/googleanalytics/_meta.json

```diff
@@ -0,0 +1,11 @@
+{
+  "overview": "Overview",
+  "configuration": "Configuration",
```

Comment:
```suggestion
  "configuration": "Configuration",
  "authentication": "Authentication,
```

Otherwise it comes after tables:
![image](https://user-images.githubusercontent.com/26760571/224013356-1764b3a2-b66d-40e7-9bd6-c38a6dd6c508.png)

Reply:
I don't understand where my change go. Was it overwritten somehow?